google/ggrc-core
# GGRC-8642 Replace can-map with can-define-map for COMMENTS \ URLS \ FILES

# Issue description

Actually can-map, can-list are used. can-map, can-list are legacy and the alternatives in latest versions of CanJS are DefineMap, DefineList.

# Steps to test the changes

Need to check:
- comments (comments list, add comments function)
- URLs (add, remove, urls list)
- Files and Folders (add, change, remove, files list, folder)

# Sanity checklist




# PR Review checklist



###
